src/main/resources/migrations/20190415_users_and_availabilities.xml

```diff
@@ -0,0 +1,51 @@
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+    <changeSet id="20190415_users_and_availabilities" author="Kay">
+
+        <!-- users -->
+        <createTable tableName="users">
+            <column name="id" type="bigint" autoIncrement="true">
```

Comment:
As discussed, we can use [HashIds](https://hashids.org) to prevent leakage of internal ids.
I am in favor of using integers, because:

- no need to map between internal and external ids, simplifies queries
- easier and faster pagination, avoiding sql `offset` which can be slow on big tables
- smaller than `UUID`

Otherwise, lets wait for the PO to decide.

Reply:
@AnastasiiaL What's your opinion?